.github/workflows/manylinux.yml

```diff
@@ -156,9 +156,16 @@ jobs:
       with:
         name: wheelhouse
         path: wheelhouse
-    - name: Install the wheels previously generated
+    - name: Install the best wheel among the previously generated
       run: |
-        "${PYTHON_EXECUTABLE}" -m pip install --prefer-binary --find-links=wheelhouse jiminy_py
+        mkdir -p "$RootDir/dist"
+        cd "$RootDir/dist"
+        # extract the "best" wheel for our system out of the wheelhouse, using pip's own resolution
+        # we cannot directly install with find-links because it does not guarantee precedence of local wheels
+        # see https://github.com/pypa/pip/issues/2589 (pip>=21.1)
+        "${PYTHON_EXECUTABLE}" -m pip download --no-index --find-links="$RootDir/wheelhouse" --no-deps jiminy_py
```

Comment:
What about using `--dest="$RootDir/dist"` and remove both `mkdir` and `cd` ?
(and replace `jiminy_py*` by `$RootDir/dist/*` of course)